cloudsync/long_poll.py

```diff
@@ -92,3 +92,7 @@ def do(self):  # this is really "do_once"
                     self.last_set = time.monotonic()
                 log.exception('Unhandled exception during long poll %s', e)
                 Runnable.backoff()
+
+    def stop(self):
+        # Could be stuck waiting for up to long_poll_timeout seconds
```

Comment:
to clarify, this happens if wait=True, and is the reason we set wait=False here?

Reply:
That's correct. Will clarify